playground/docker-compose.fork.yml

```diff
@@ -123,10 +126,15 @@ services:
     environment:
       - ETHRPC=http://chain:8545
       - ADDR=0.0.0.0:80
+      - RUST_BACKTRACE=1
```

Comment:
`RUST_BACKTRACE` can also be added to the orderbook, autopilot and solver.

Reply:
We should keep `TOML_TRACE_ERROR` here. It makes it easier to debug when you have parsing issues with your config file.